chpl/chpl-service/src/main/java/gov/healthit/chpl/web/controller/SurveillanceController.java

```diff
@@ -668,4 +680,19 @@ private void checkUploadedSurveillanceOwnership(Surveillance pendingSurv) {
 			} 
 		}
 	}
+	
+	private void updateAuthority(Surveillance surv){
```

Comment:
Since I believe the UI will not allow the user to update the authority, I think I can remove the call to updateAuthority() from the /update method. 

For the create method, the API controller requires a RequestBody with a Surveillance object, but I don't think it will catch if one of the Surveillance properties is null--that's why we have the validators, right? I thought the requirement in OCD-1250 stated that if the request has a null ROLE, then to use the role from the User object. Thus, we shouldn't deny the request from being accepted. Am I misunderstanding something?